src/cli/cli_br.hpp

```diff
@@ -76,9 +76,16 @@ class Br : private Output
 private:
     using Command = CommandEntry<Br>;
 
+    using PrefixType = uint8_t;
+    enum : PrefixType
+    {
+        kPrefixTypeLocal   = 0x01,
+        kPrefixTypeFavored = 0x02,
```

Comment:
This is ok as is, but we can use  `1u << 0` and `1u << 1` to reinforce that they are bitmask?